docs/src/main/paradox/r2dbc.md

```diff
@@ -43,7 +47,7 @@ The table below shows `akka-projection-r2dbc`'s direct dependencies, and the sec
 ## Schema
 
 The `akka_projection_offset_store`, `akka_projection_timestamp_offset_store` and `akka_projection_management` tables
-need to be created in the configured database:
+need to be created in the configured database, unless using H2 where the schema is created automatically:
```

Comment:
Wouldn't they have to define the tables in `additional-init` of their application.conf?

Could we define the default h2 tables in reference conf in something like
```
akka.projection.r2dbc.h2.default-additional-init = """...
```

and then they would only have to include that
```
additional-init = ${akka.projection.r2dbc.h2.default-additional-init}
```

Reply:
Ah, I missed this part, we have it in `application-h2.conf` for the tests, not integrated in some clever way so users do not have to deal with it yet (but we should)

Reply:
Pro-tip: Imagine you did something clever and then document that thing that you never actually implemented 🤦 

Fixed in 1c9a3e6f6af5bd7f94951701b9d0e9d32aea66fc